sql/CMakeLists.txt

```diff
@@ -804,7 +804,7 @@ TARGET_LINK_LIBRARIES(sql_main ${MYSQLD_STATIC_PLUGIN_LIBS}
 # sql/immutable_string.h uses
 # google::protobuf::io::CodedOutputStream::WriteVarint64ToArray
 # which may or may not be inlined.
-TARGET_LINK_LIBRARIES(sql_main ${PROTOBUF_LITE_LIBRARY})
+#TARGET_LINK_LIBRARIES(sql_main ${PROTOBUF_LITE_LIBRARY})
```

Comment:
Please add a comment on why we not linking PROTOBUF. So that future merges we don't uncomment by mistake.

Reply:
may be remove the line

Reply:
removed